src/content/docs/cypher/expressions/node-rel-functions.md

```diff
@@ -47,13 +48,23 @@ MATCH (a) RETURN LABEL(a) AS LABEL LIMIT 1;
 ```
 Output:
 ```
----------
-| LABEL |
----------
-| User  |
----------
+┌────────┐
+│ LABEL  │
+│ STRING │
+├────────┤
+│ User   │
+└────────┘
 ```
 
+:::note[Alias `LABELS` is available]
+As of version 0.9.0, the `LABELS` function can also be used as an alias for the `LABEL` function.
+For example:
+
+```cypher
+MATCH (a) RETURN LABELS(a) AS LABELS LIMIT 1;
+```
+:::
+
```

Comment:
Do we really need to highlight this?

As a general rule, I think docs should have a single "best practice" (ref my other comment)

Reply:
Maybe not, but I want to bring it to the forefront for the short term because this is a Neo4j function that people can come to expect when then use Kuzu. We can remove the emphasis in a future version.

Reply:
I feel this belongs more to a changelog or announcement post than the documentation page. But okay for now.